### ISSUE TITLE
Make more SetFitClassification parameters configurable

### ISSUE THREAD

**Issue by chschroeder** [webis-de/small-text#46]
## Feature description

There are still important parameters which cannot be easily configured. The following list of parameters should be configurable via `SetFitClassification` / `SetFitModelArguments`:

- SetFitTrainer
  - learning_rate
  - num_iterations
  - warmup_proportion

This list is intentionally not yet exhaustive, since especially with different loss functions configuration could get messy, which I want to avoid. The parameter `amp` would also be included, however, this is already included in #45.

## Motivation

-

## Additional comments

-
